multiqc/utils/search_patterns.yaml

```diff
@@ -279,6 +279,10 @@ gatk/varianteval:
 gatk/base_recalibrator:
     contents: '#:GATKTable:Arguments:Recalibration'
     num_lines: 3
+gffcompare:
+    fn: '*.stats'
```

Comment:
```suggestion
```

I'm guessing that this is stdout / stderr? In which case there should not be a filename pattern as it could be anything.

Reply:
Answering my own question - this is not stdout / stderr, this is a common prefix used by the tool: 
https://ccb.jhu.edu/software/stringtie/gffcompare.shtml#output-files

> Data summary and accuracy estimation: `<outprefix>.stats`

So this search pattern setup is perfect as it is. Fast filename finding but as the suffix is super generic, an additional contents check 👍🏻